lib/types.js

```diff
@@ -517,6 +510,14 @@ function disposeObjectArray () {
     }
     dispose.call(obj);
   }
+
+  const wrapper = this.$w;
+  if (wrapper !== undefined) {
+    const dispose = wrapper.$dispose;
+    if (dispose !== undefined) {
+      dispose.call(wrapper);
```

Comment:
I suppose we could use nullish coalescing syntax to simplify this?

Reply:
Yes, fixed. Also, I think using `call` here is not necessary? Personally, I did it that way for it to become the same as other calls to the same function. However, automatic binding should already take care of passing the correct `this` context in this case.